synapse/models/inet.py

```diff
@@ -330,6 +330,29 @@ def repr(self, valu):
             except UnicodeError:
                 return valu
 
+class HttpCookie(s_types.Str):
+
+    def _normPyStr(self, text):
+
+        text = text.strip()
+        parts = text.split('=', 1)
+
+        name = parts[0].split(';', 1)[0].strip()
+        if len(parts) == 1:
+            return text, {'subs': {'name': name}}
```

Comment:
There is probably more edge cases for handling Set-Cookie strings than what we cover here - it would be good to review this https://github.com/httpwg/http-extensions/issues/159 to plumb for more edge cases.

Reply:
This is a KISS use case. It's only meant to pull out name/value if they are easily available.

Reply:
( and it does handle `name` , `name=` and `name=value` )